miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultProjectService.java

```diff
@@ -182,6 +182,15 @@ private void validateChange(Project project, Project beforeChange) throws IOExce
       errors.add(new ValidationError("title", "There is already a project with this title"));
     }
 
+    List<Long> uniqueContacts = new ArrayList<>();
+    for (ProjectContact element : project.getContacts()) {
+      if (uniqueContacts.contains(element.getContact().getId())) {
+        errors.add(new ValidationError("contacts", "You cannot use the same contact twice"));
+        break;
```

Comment:
`continue` instead of `break` so that all duplicates will be reported instead of only the first